site/docs/hive.md

```diff
@@ -50,6 +50,38 @@ You should now be able to issue Hive SQL `SELECT` queries using the above table
 SELECT * from table_a;
 ```
 
+#### Using Hive Catalog
+Iceberg tables created using `HiveCatalog` are automatically registered with Hive.
+
+##### Create an Iceberg table
+The first step is to create an Iceberg table using the Spark/Java/Python API and `HiveCatalog`. For the purposes of this documentation we will assume that the table is called `table_b` and that the table location is `s3://some_path/table_b`. In order for Iceberg to correctly set up the Hive table for querying some configuration values need to be set, the two options for this are described below - you can use either or the other depending on your use case.
+
+##### Hive Configuration
+The value `iceberg.engine.hive.enabled` needs to be set to `true` and added to the Hive configuration file on the classpath of the application creating the table. This can be done by modifying the relevant `hive-site.xml`. Alternatively this can done programatically like so:
```

Comment:
I assume we can query existing Iceberg tables also through Hive by setting the `iceberg.engine.hive.enabled` flag? Do you think it worth calling that out?

Reply:
I'll let you know if and when I get it working ;) and yes, that will need to be added to the documentation.

Reply:
Updated docs to describe the property you need to set for Hive read is `set iceberg.mr.catalog=hive`